src/Umbraco.Cms.Targets/Umbraco.Cms.Targets.csproj

```diff
@@ -58,4 +60,9 @@
       <_PackageFiles Include="$(IntermediateOutputPath)_._" PackagePath="lib\$(TargetFramework)" />
     </ItemGroup>
   </Target>
+
+  <Target Name="GenerateUmbracoPackageSchema" BeforeTargets="Build;CopyUmbracoJsonSchemaFiles" Condition="!Exists('$(_UmbracoCmsPackageSchemaReference)')">
+    <Message Text="Generating $(_UmbracoCmsPackageSchemaReference) because it doesn't exist" Importance="high" />
+    <Copy SourceFiles="$(MSBuildThisFileDirectory)..\Umbraco.Web.UI.New.Client\dist-cms\umbraco-package-schema.json" DestinationFiles="$(_UmbracoCmsPackageSchemaReference)" />
```

Comment:
That is really the big question if we need to run the command. The schema file is generated when the backoffice is being built and will not change from that point no matter how many times you run this command. I can change this to actually run the command, but it will then be generated twice. What do you think?

Reply:
The backoffice is also located in static assets, so if you can find the file from there it is okay, but the other projects do not work, as they are not packable, and not build doing the pack. I also gust get an error about the file is missing when trying it out.

Reply:
I get this error `  Umbraco.Cms.Targets.csproj(66, 5): [MSB3030] Could not copy the file "/Users/berg/RiderProjects/Umbraco-CMS-V14/src/Umbraco.Cms.Targets/../Umbraco.Web.UI.New.Client/dist-cms/umbraco-package-schema.json" because it was not found.` on my machine

Reply:
Should work now